content/en/docs/setup/upgrade/index.md

```diff
@@ -90,6 +92,30 @@ $ istioctl proxy-config endpoints ${pod_name}.test-ns --cluster xds-grpc -ojson
 
 The output confirms that the pod is using `istiod-canary` revision of the control plane.
 
+### Uninstall old control plane
+
+After upgrading both the control plane and data plane, you can uninstall the old control plane. For example, the following command uninstalls a control plane of revision `istio-1-6-5`:
+
+{{< text bash >}}
+$ istioctl x uninstall --revision istio-1-6-5
+{{< /text >}}
+
+If you previously installed Istio with customized installation options, you can run the following command:
+
+{{< text bash >}}
+$ istioctl x uninstall <your original installation options>
+{{< /text >}}
+
```

Comment:
This second option seems confusing. Why is it needed? Aren't the options available in the in-cluster CR?

Reply:
What I wanted to mention is that users can also uninstall with the generated manifest from the original install option, for example if they previous installed istio with `istioctl install -f iop.yaml --set foo=bar` then `istioctl x uninstall -f iop.yaml --set foo=bar` would remove the generated manifests from the cluster.

Reply:
To avoid confusion, I moved this part to the ##uninstall istio section in the istioctl guide instead.